src/platform/Tizen/ConnectivityManagerImpl.cpp

```diff
@@ -224,6 +253,56 @@ bool ConnectivityManagerImpl::IsWiFiManagementStarted()
     return isActivated;
 }
 
+CHIP_ERROR ConnectivityManagerImpl::GetWiFiBssId(ByteSpan & value)
+{
+    uint8_t bssId[kMaxHardwareAddrSize];
+    CHIP_ERROR err = Internal::WiFiMgr().GetBssId(bssId);
+    VerifyOrReturnError(err == CHIP_NO_ERROR, err);
+
+    value = ByteSpan(bssId, 6);
+
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR ConnectivityManagerImpl::GetWiFiSecurityType(SecurityTypeEnum & securityType)
+{
+    wifi_manager_security_type_e secType;
+    CHIP_ERROR err = Internal::WiFiMgr().GetSecurityType(&secType);
+    VerifyOrReturnError(err == CHIP_NO_ERROR, err);
+
+    switch (secType)
+    {
+    case WIFI_MANAGER_SECURITY_TYPE_NONE:
+        securityType = SecurityTypeEnum::kNone;
+        break;
+    case WIFI_MANAGER_SECURITY_TYPE_WEP:
+        securityType = SecurityTypeEnum::kWep;
+        break;
+    case WIFI_MANAGER_SECURITY_TYPE_WPA_PSK:
+        securityType = SecurityTypeEnum::kWpa;
+        break;
+    case WIFI_MANAGER_SECURITY_TYPE_WPA2_PSK:
+        securityType = SecurityTypeEnum::kWpa2;
+        break;
+    case WIFI_MANAGER_SECURITY_TYPE_EAP:
+    case WIFI_MANAGER_SECURITY_TYPE_WPA_FT_PSK:
+    case WIFI_MANAGER_SECURITY_TYPE_SAE:
+    case WIFI_MANAGER_SECURITY_TYPE_OWE:
+    case WIFI_MANAGER_SECURITY_TYPE_DPP:
+    default:
+        securityType = SecurityTypeEnum::kUnspecified;
+        break;
+    }
+
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR ConnectivityManagerImpl::GetWiFiVersion(WiFiVersionEnum & wiFiVersion)
+{
+    wiFiVersion = WiFiVersionEnum::kN;
```

Comment:
This is the only version supported?

Also, that's not the right return value if wifi is not even connected...

Reply:
It would be better to modify it to unimplemented. i'll update it.